packages/ai/src/Button.ts

```diff
@@ -40,14 +47,36 @@ import styles from "./generated/themes/Button.css.js";
  * @since 2.0
  * @public
  */
+
 @customElement({
 	tag: "ui5-ai-button",
+	languageAware: true,
 	renderer: litRender,
-	template,
-	styles,
-	dependencies: [MainButton],
+	template: ButtonTemplate,
+	styles: ButtonCss,
+	dependencies: [MainButton, Icon, ButtonState],
+	shadowRootOptions: { delegatesFocus: true },
 })
+
+/**
+ * Fired when the component is activated either with a
+ * mouse/tap or by using the Enter or Space key.
+ * @public
+ */
+@event("click")
+
+/**
+ * @experimental
+ */
 class Button extends UI5Element {
+	/**
+	 * Defines the component design.
+	 * @default "Default"
+	 * @public
+	 */
+	@property({ type: ButtonDesign, defaultValue: ButtonDesign.Default })
+	design?: `${ButtonDesign}`;
```

Comment:
Should we support this? Just asking, i haven't looked at the spec, but as far as i can remember I didn't see different designs of the AI Button.

Reply:
Yes, we should, somewhere in the Figma pages I have noticed Emphasized AI button, but didn't remember where exactly.